docs/examples/crawler_stop.mdx

```diff
@@ -0,0 +1,15 @@
+---
+id: crawler-stop
+title: Crawler stop
```

Comment:
Can we use something better? e.g. "Stopping a Crawler with stop method", or "Halt the crawling with stop method", ...

Reply:
Done